ref/src/yppm_core.F90

```diff
@@ -21,11 +21,10 @@
 
 module yppm_core_mod
 
-  implicit none
-
-  private
+  use netCDFModule
+  use interpolate
 
-  public yppm
```

Comment:
This isn't needed because you removed the `private` above.  However, I'm not sure that was wise.